handbook/engineering/deployments/playbooks.md

```diff
@@ -1,16 +1,17 @@
 # Playbooks for deployments
 
 - [Playbooks for deployments](#playbooks-for-deployments)
-  - [Sourcegraph.com](#sourcegraphcom)
-    - [Deploying to sourcegraph.com](#deploying-to-sourcegraphcom)
-    - [Rolling back sourcegraph.com](#rolling-back-sourcegraphcom)
-    - [Invalidating all user sessions](#invalidating-all-user-sessions)
-    - [Accessing sourcegraph.com database](#accessing-sourcegraphcom-database)
-      - [Via the CLI](#via-the-cli)
-      - [Via BigQuery (for read-only operations)](#via-bigquery-for-read-only-operations)
-  - [k8s.sgdev.org](#k8ssgdevorg)
-    - [Manage users in k8s.sgdev.org](#manage-users-in-k8ssgdevorg)
-  - [Cloudflare Configuration](#cloudflare-configuration)
+    - [Sourcegraph.com](#sourcegraphcom)
+        - [Deploying to sourcegraph.com](#deploying-to-sourcegraphcom)
+        - [Rolling back sourcegraph.com](#rolling-back-sourcegraphcom)
+        - [Invalidating all user sessions](#invalidating-all-user-sessions)
+        - [Accessing sourcegraph.com database](#accessing-sourcegraphcom-database)
+            - [Via the CLI](#via-the-cli)
+            - [Via BigQuery (for read-only operations)](#via-bigquery-for-read-only-operations)
+        - [Restarting docs.sourcegraph.com](#restarting-about-sourcegraph-com-and-docs-sourcegraph-com)
+    - [k8s.sgdev.org](#k8ssgdevorg)
+        - [Manage users in k8s.sgdev.org](#manage-users-in-k8ssgdevorg)
+    - [Cloudflare Configuration](#cloudflare-configuration)
```

Comment:
This list wasn't rendering correctly due to https://github.com/russross/blackfriday/issues/329, so I've reindented it to nest it properly. Here's how it looked:

![image](https://user-images.githubusercontent.com/229984/110704319-a7d2d380-81a9-11eb-90e4-4c205bda6694.png)

Reply:
Nice!